app/bundles/CampaignBundle/Tests/Command/ExecuteEventCommandTest.php

```diff
@@ -58,12 +63,132 @@ public function testEventsAreExecutedForInactiveEventWithSingleContact(): void
 
                 continue;
             }
-
-            if (1 === (int) $log['is_scheduled']) {
-                $this->fail('Event is still scheduled for lead ID '.$log['lead_id']);
-            }
```

Comment:
Is this failing? I'm worried that if it is failing then this PR is breaking an expected behavior.

Reply:
I fixed this in https://github.com/mautic/mautic/pull/13263/commits/ea4a17f5754bf3cb4593018b859c55bb691bc298. It took quite a lot of time to figure out what's wrong and how to fix it. But removing a test assertion because it started to fail after some changes were made is really bad.